pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -241,11 +241,9 @@ func extractFiles(archiveFile string, pluginName string, filePath string, allowS
 					continue
 				}
 			} else {
-
 				err = extractFile(zf, newFile)
 				if err != nil {
-					logger.Errorf("Failed to extract file: %v \n", err)
-					continue
+					return errutil.Wrap("Failed to extract file", err)
```

Comment:
@aocenas are you aware of any particular reason we don't want to abort installing plugin when extraction fails? Do you see any risk with adding this?

Reply:
Maybe if we can be semi-successful? We still probably want to abort, but abort entirely rather than partially.

Reply:
Full failure is much better.  Should we also try to delete anything we did extract?  A halfway extracted plugin will just confuse people

Reply:
Don't think I made the initial decision for this code so only thing that I see is that this is per file error so it may be better for debugging to collect all the files that have problems with extraction and then show all the errors afterwards and failing hard if there is more than 0 file errors.

Reply:
@ryantxu that feels like a new issue/PR. Regarding delete anything we did extract. With current solution we would try to delete the failed extracted plugin and/or an existing installed plugin and that operation will also fail if a backend plugin is in use. . 

One thing we need would be to extract all files to a temporary directory to verify extraction was successful, before trying to overwrite an existing installed plugin. Need to create an issue for this.

Highly related to backend plugins moving forward. Will add this to Backend Plugin API Roadmap.

@aocenas Thanks. Will consider collecting all files/problem in enhancement of this, but will merge this fix for now.